samples/sample_common/include/sample_defs.h

```diff
@@ -145,7 +145,7 @@ enum LibVABackend
 #define MSDK_CHECK_NOT_EQUAL(P, X, ERR)          {if ((X) != (P)) {msdk_stringstream tmpStr3;tmpStr3<<MSDK_STRING(#X)<<MSDK_STRING("!=")<<MSDK_STRING(#P)<<MSDK_STRING(" error"); \
     MSDK_PRINT_RET_MSG(ERR, tmpStr3.str().c_str()); return ERR;}}
 
-#define MSDK_CHECK_STATUS(X, MSG)                {if ((X) < MFX_ERR_NONE) {MSDK_PRINT_RET_MSG(X, MSG); return X;}}
+#define MSDK_CHECK_STATUS(X, MSG)                {const bool sts_is_error = (X) < MFX_ERR_NONE; if (sts_is_error) {MSDK_PRINT_RET_MSG(X, MSG); return X;}}
```

Comment:
@egorovdanil : we talked offline w/ @onabiull and @mgonchar. We don't like this workaround either. Let's shut up MSFT compiler warning with conditional if to avoid clang cry out. I.e.:
```
#ifdef _MSC_VER
#pragma warning(disable: XXX)
#endif 
```

Reply:
done